packages/block-editor/src/components/block-list/index.js

```diff
@@ -38,14 +38,21 @@ function BlockList(
 		const {
 			getBlockOrder,
 			getBlockListSettings,
+			getSettings,
 			getSelectedBlockClientId,
 			getMultiSelectedBlockClientIds,
 			hasMultiSelection,
 			getGlobalBlockCount,
 			isTyping,
 			isDraggingBlocks,
+			__experimentalGetActiveBlockIdByBlockNames,
 		} = select( 'core/block-editor' );
 
+		// Determine if there is an active entity area to spotlight.
+		const activeEntityBlockId = __experimentalGetActiveBlockIdByBlockNames(
+			getSettings().__experimentalSpotlightBlocks
```

Comment:
Would it make sense to make this a default parameter value if it's not passed? Then we could also probably shorten the name of the selector to `__experimentalGetActiveEntityBlockId`, which better aligns it with surrounding variable names that we are using. I guess it depends on how often we anticipate this selector will be used outside of this functionality.

Also I'd probably rename `__experimentalSpotlightBlocks` to `__experimentalSpotlightEntityBlocks` to avoid potential confusion with regular spotlight feature.

Reply:
> Would it make sense to make this a default parameter value if it's not passed?
> I guess it depends on how often we anticipate this selector will be used outside of this functionality.

Yeah, I guess that 100% depends on how this selector might be used.  For our purposes, that would make sense!  However, without that change this selector is more general and could be used for other purposes.  Whether it would be, I really don't know. 🤷‍♀️ 

> Also I'd probably rename __experimentalSpotlightBlocks to __experimentalSpotlightEntityBlocks to avoid potential confusion with regular spotlight feature.

👍